ko/lessons/basics/documentation.md

```diff
@@ -23,7 +23,7 @@ Elixir에서는 문서화를 *일급 시민*으로 취급하고 있습니다. 
 
 ### 인라인 문서화
 
-코드에 주석을 다는 가장 심플한 방법은 인라인 주석일 겁니다. Ruby, Python과 비슷하게 Elixir의 인라인 주석은 `#`로 나타낼 수 있습니다. 이는 환경에 따라 다르지만 우물정자 또는 해시 로 알려져 있기도 합니다. 
+코드에 주석을 다는 가장 심플한 방법은 인라인 주석일 겁니다. Ruby, Python과 비슷하게 Elixir의 인라인 주석은 `#`로 나타낼 수 있습니다. 이는 환경에 따라 다르지만 우물정자 또는 해시 로 알려져 있기도 합니다.
```

Comment:
> Similar to Ruby or Python, Elixir's inline comment is denoted with a `#`, frequently known as a *pound*, or a *hash* depending on where you are from in the world.

Original was pound so, I think 파운드 or 우물 정 will fine here.
I tread "우물 정자" bad word. since, it  can misread as pound Chinese pavilion.

<https://ko.wikipedia.org/wiki/%ED%95%B4%EC%8B%9C_%EA%B8%B0%ED%98%B8>